great-idea/css/index.css

```diff
@@ -63,4 +63,102 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/*START >>  */
+body{
+	max-width:1024px;
+	margin:40px 100px 40px 100px;
+}
+.nav-container{
+	width:100%;
+	margin-bottom:75px;
+	display:flex;
+}
+.nav-container nav, img{
+	display:flex;
+}
+.nav-container nav{
+	display:flex;
+	align-items:center;
+}
+nav a {
+	margin:0 40px 0 0;
```

Comment:
If you're only putting margin on one side you can specify the side `margin-right: 40px`